mysql-test/suite/rocksdb/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --explicit-defaults-for-timestamp=1 --loose-rocksdb_lock_wait_timeout=1 --loose-rocksdb_strict_collation_check=0 --sql_mode=NO_ENGINE_SUBSTITUTION
```

Comment:
Yup, precisely, and why I logged https://jira.percona.com/browse/MYR-110 as this is an unacceptable setting, but, out of the scope of this change to fix as it will likely require the re-writing and/or re-recording of much of the suits.